lib/beaker/host.rb

```diff
@@ -438,7 +438,7 @@ def do_scp_to source, target_path, options
           @logger.trace result.stdout
         end
       else # a directory with ignores
-        dir_source = Dir.glob("#{source}/**/*").reject do |f|
+        dir_source = Dir.glob("#{source}/**/{*,.*}").reject do |f|
```

Comment:
I gave it a go in the next commit.
However, the spec tests pass with both versions of the code, which is probably due to FakeFS implementation in respect to `glob()`'s behavior.
Unfortunately, getting the acceptance suite going was a little outside of scope of my weekend endeavour, but the test should look similar to this.